pkg/webhook/policy.go

```diff
@@ -62,7 +63,7 @@ import (
 // https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#response
 const httpStatusWarning = 299
 
-var maxServingThreads = flag.Int("max-serving-threads", -1, "(alpha) cap the number of threads handling non-trivial requests, -1 means an infinite number of threads")
+var maxServingThreads = flag.Int("max-serving-threads", -1, "cap the number of threads handling non-trivial requests, -1 caps the number of threads to GOMAXPROCS. Defaults to -1.")
```

Comment:
In general, what is the criteria to graduate a flag from alpha to beta, then removing the status?

Reply:
Do we need to add anything about how GOMAXPROCS is determined? If not here, maybe in docs?

Reply:
I think there are two criteria:

1) we are confident it's the right flag (we won't remove it, change it, its behavior, or what is passed to it in a backwards incompatible way -- though I don't know how strongly we want to guarantee that)

2) We don't want to signal risk

Given this flag has existed for a while, I don't think we need to worry about either here.


We can talk about GOMAXPROCS some I guess? Golang has some default logic, though we are overriding that with automaxprocs. Is that a blocker for this PR?

Reply:
> Golang has some default logic, though we are overriding that with automaxprocs. Is that a blocker for this PR?

Not a blocker. Just thought it's easier to do it now so we don't forget to update the docs.

Reply:
Ack. Added a doc on performance tuning.